# Initial States #

# Position
x_pos: 0
y_pos: 0
z_pos: 0

# Orientation
roll: 0
pitch: 0
yaw: 0

# Number of particles
particles: 500

# Standard deviations for movement model
/movement/x_std_dev: 0.2
/movement/y_std_dev: 0.2
/movement/z_std_dev: 0.2

/movement/roll_std_dev: 0.2
/movement/pitch_std_dev: 0.2
/movement/yaw_std_dev: 0.2

# Frames
worldFrame: "world"
baseFootprintFrame: "base_footprint"
baseStabilizedFrameID: "base_stabilized"
baseLinkFrameID: "base_link"

# Observation parameters / Raycasting
laser_z_hit: 0.5
laser_z_short: 0.05
laser_z_rand: 0.5
laser_z_max: 0.05
laser_sigma_hit: 0.2
laser_lambda_short: 0.1
min_range: 0.01
max_range: 14
